ServerCore/Pages/Puzzles/Play.cshtml

```diff
@@ -43,9 +43,18 @@
        asp-route-teamPuzzleSort="@Model.TeamPuzzleSort"
        asp-route-stateFilter=@(unsolvedFilter ? PlayModel.PuzzleStateFilter.All : PlayModel.PuzzleStateFilter.Unsolved)>
         Show @(unsolvedFilter ? "All" : "only unsolved") puzzles
-    </a> |
-    <a asp-page="/Threads/PuzzleThreads">View help threads</a>
+    </a> | 
+    <a asp-page="/Threads/PuzzleThreads">View help threads</a> | 
```

Comment:
What is the importance of this change? Doesn't seem bug bash motivated so I'm wondering if it's an intentional addition.

Reply:
This is general page cleanup that falls under the "make the site look better" feedback.